api/src/main/java/ai/djl/inference/Predictor.java

```diff
@@ -284,6 +281,18 @@ public NDManager getNDManager() {
             return ctxManager;
         }
 
+        /** {@inheritDoc} */
+        @Override
+        public NDManager getPredictorManager() {
```

Comment:
Maybe just `getManager`?

Reply:
Nevermind. This is on context rather than predictor so the clarification is helpful.